core/openapi/schemas/party.yaml

```diff
@@ -1,103 +1,89 @@
-                properties:
-                  deliveryPoint:
-                    type: string
-                    description: Address line for the location.
-                  city:
-                    type: string
-                    description: City for the location.
-                  administrativeArea:
-                    type: string
-                    description: State or province of the location.
-                  postalCode:
-                    type: string
-                    description: ZIP or other postal code.
-                  country:
-                    type: string
-                    description: Country of the physical address.
-          url:
-            $ref: common/link.yaml
-            description: On-line information about the responsible party.
-          hoursOfService:
+        description: Telephone numbers at which contact can be made.
+        patternProperties:
+          "^.*":
             type: string
             description: |-
-              Time period (including time zone) when the resposible party
-              can be contacted.
-          contactInstructions:
+              The key name indicates the type of phone number (e.g. home,
+              work, fax, etc.). The value is the phone number itself.
+            pattern: "^+[1-9]{1}[0-9]{3,14}$"
+      email:
+        type: object
+        description: Email address at which contact can be made.
+        patternProperties:
+          "^.*":
             type: string
             description: |-
-              Supplemental instructions on how or when to contact the
-              responsible party.
-      roles:
-        $ref: roles.yaml
+              The key name indicates the type of email address (e.g. home,
+              work, etc.). The value of the email address itself.
+            format: email
+      address:
+        type: object
+        description: Physical location at which contact can be made.
+        patternProperties:
+          "^.*":
+            type: object
+            description: |-
+              The key name indicates the type of address (e.g. office,
+              home, etc.). The value is the address itself.
+            properties:
+              deliveryPoint:
+                type: string
+                description: Address line for the location.
+              city:
+                type: string
+                description: City for the location.
+              administrativeArea:
+                type: string
+                description: State or province of the location.
+              postalCode:
+                type: string
+                description: ZIP or other postal code.
+              country:
+                type: string
+                description: Country of the physical address.
+      url:
+        $ref: common/link.yaml
+        description: On-line information about the responsible party.
+      hoursOfService:
```

Comment:
I don't think a free-text is very useful here. Sure, you can add everything, but it's not machine readable at all. I'd prefer to either remove this or make it more structured. Thoughts?

Reply:
@m-mohr threw together a schema for HOS.  Not sure I did it correctly or usefully but the idea is the be able to say something like:
````
hoursOfService: {
   "1": [ ["9:00","17:00"]], 
   "3": [["9:00","17:00"]],
   "4": [["10:00","12:00"],["13:00","15:30"]],
   "closed": ["-12-25","-07-20"]
}
````
Please review and mark as resolved if you are OK with the changes.

Reply:
fwiw in pygeoapi we've loosely aligned with https://schema.org/openingHours

Reply:
@tomkralidis, yeah, saw that.  Was not crazy about it.  Didn't think it was granular enough but if people like that instead, we can go with that ... or remove HOS altogether.  Do people really use this?

Reply:
We use this in MSC.

Reply:
@pvretano @m-mohr looking at this again, isn't`hoursOfService` as  a `string` enough here?  I'm just concerned that providers will have issues producing a valid definition.  Yes, a text string is relativiely free form and perhaps not as machine readable, but it is human readable and fits the use case of review/assessment of metadata when looking to get in touch with a contact.

Reply:
Yeah, this was merged rather quickly. I think the current definition is not ideal ("1sad" is a valid key, isn't it?), but a string is also not ideal. I feel like hoursOfService is so niche, that I'd actually remove it as it can also be encoded in the free-text field `contactInstructions` as it is describes as "[...] instructions on [...] when to contact the responsible party."

Reply:
Created #245 for it. Maybe you want to add your comment there, as well? @tomkralidis